.pre-commit-config.yaml

```
@@ -1,17 +1,28 @@
 repos:
 -   repo: https://github.com/asottile/reorder_python_imports
-    rev: v1.4.0
+    rev: v2.3.5
     hooks:
     -   id: reorder-python-imports
 -   repo: https://github.com/ambv/black
-    rev: 18.6b4
+    rev: 20.8b1
     hooks:
     -   id: black
         args: [--safe, --quiet]
         language_version: python3
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v1.3.0
+    rev: v3.3.0
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
     -   id: debug-statements
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.4
+    hooks:
+    -   id: flake8
+        additional_dependencies:
+            - pep8-naming
+            - flake8-typing-imports==1.9.0
+        language_version: python3
+        args:
+            - --ignore=E,W,N803
```

```suggestion
            - "--ignore=E,W,N803"
```

Not required but I find this reads better due to the leading `--`